conf/defaults.ini

```diff
@@ -379,16 +379,32 @@ send_client_credentials_via_post = false
 
 #################################### SAML Auth ###########################
 [auth.saml] # Enterprise only
-enabled = false
-private_key =
-private_key_path =
+# Defaults to false. If true, the feature is enabled.
```

Comment:
these comments should be in sample.ini (more important there as that is the grafana.ini file that gets copied to /etc/grafana/grafana.ini on first install)

Reply:
Great shout.